src/templates/app/campus-group-page.js

```diff
@@ -2,7 +2,7 @@
 
 import React from 'react'
 import { useQuery } from '@apollo/react-hooks'
-import CampusGroupAction from '../../components/CampusGroupAction'
+// import CampusGroupAction from '../../components/CampusGroupAction'
```

Comment:
@duranmla why commenting this line?

Reply:
Those components doesn't exist within our project, they doesn't throw runtime errors because we are not using those files, but they do throw errors with flow that yields the fact this component doesn't exists.

Reply:
![image](https://user-images.githubusercontent.com/1425162/72414615-f128be00-3772-11ea-97ca-4007b0f8613c.png)

Reply:
if it's safe to delete those files I think we should do that. @duranmla